tests/hil/scripts/pytest-hil/plugin.py

```diff
@@ -29,9 +33,18 @@ def baud(request):
 def credentials_file(request):
     return request.config.getoption("--credentials-file")
 
+@pytest.fixture(scope="session")
+def hex_file(request):
+    return request.config.getoption("--hex-file")
+
+@pytest.fixture(scope="session")
+def snr(request):
+    return request.config.getoption("--snr")
```

Comment:
Nit. I think we could be more specific and not abbreviate too much here. `snr` is `serial_number` in this case, so I would prefer the latter.